### PR TITLE
Removing SideMenu from Patient Chart app

### PR DESCRIPTION
**Pull request by vasharma05**
**What does this PR do?**
I have removed the Side Menu component from the Patient Chart App which was conflicting with the side menu present in the Primary Navigation and is also visible in the desktop layout. The before/after images are attached for the reference.
Before:
image
After:
image
 
P.S.: The major changes are in the Patient-chart.component.tsx and it's corresponding patient-chart.component.scss, all the other changes were the errors given by Prettier, which were solved by running `npm run prettier` in the base monorepo.

### CODE REVIEW

File: packages/esm-patient-chart-app/src/ui-components/patient-chart/patient-chart.component.scss [hunk 1]
```diff
@@ -35,10 +35,6 @@
   flex-direction: column;
 }
 
-:global(.omrs-breakpoint-gt-tablet) .chartContainer {
-  margin-left: var(--omrs-sidenav-width);
-}
-
```
Comment (vasharma05) [inline packages/esm-patient-chart-app/src/ui-components/patient-chart/patient-chart.component.scss:41]: @FlorianRappl , this is the global prop I removed that gave the left margin to the patient-chart page.

File: packages/esm-patient-chart-app/src/ui-components/patient-chart/patient-chart.component.tsx [hunk 2]
```diff
@@ -27,37 +27,34 @@ const PatientChart: React.FC<RouteComponentProps<PatientChartParams>> = ({
   useVisitDialog(patientUuid);
 
   return (
-    <>
-      <SideMenu />
```
Comment (vasharma05) [inline packages/esm-patient-chart-app/src/ui-components/patient-chart/patient-chart.component.tsx:31]: @FlorianRappl This is the Side Menu component that I removed from the patient-chart-component.

File: packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx [hunk 3]
```diff
@@ -27,15 +27,15 @@ interface PatientBannerProps {
...
+  patientUuid
 }) => {
   const { t } = useTranslation();
   const { currentVisit } = useVisit(patientUuid);
   const [showContactDetails, setShowContactDetails] = useState(false);
   const [hasActiveVisit, setActiveVisit] = useState(false);
   const state = useMemo(() => ({ patientUuid }), [patientUuid]);
   const toggleContactDetails = useCallback(
-    () => setShowContactDetails((value) => !value),
+    () => setShowContactDetails(value => !value),
```
Comment (nmalyschkin) [inline packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx:38]: Could you please check, that you have the newest version of eslint/tslint/typescript installed. The formatting changes seem to be a step backwards.

**Review by nmalyschkin — COMMENTED** [review]
The many formatting changes make it unnecessary tedious to review.

File: packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx (hunk 3 above)
Comment (vasharma05) [reply packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx:38]: @nmalyschkin  Actually when I committed and the pre-commit tests ran, the prettier package gave me errors in all these files, which I solved by running `npm run prettier`.

What should I do now to undo all the prettier changes? Should I close this one and create a new PR?

**Comment by FlorianRappl** [comment]
@vasharma05 if your screenshots tell the truth then this PR needs to be closed. In the desktop view we should see the menu.

**Comment by vasharma05** [comment]
Ohh, okay so it was there on purpose. It was not before so I thought it's a bug pending. I'll just ask from the next time before working on something.
Thanks @FlorianRappl

**Comment by vasharma05** [comment]
@FlorianRappl , the package.json is having the older version of prettier 1.9, and hence I had these prettier errors, so in the last commit above, I updated the package and ran prettier, so should I push the new prettier package version and the corresponding file changes?

**Comment by nmalyschkin** [comment]
@vasharma05 I'm already updating prettier and the plugins for that.

**Comment by vasharma05** [comment]
Thanks @nmalyschkin !

**Comment by FlorianRappl** [comment]
Also keep in mind its a lerna monorepo. So don't use `npm` to manage dependencies. Use `lerna`. It will use the configured package manager (which in this case is Yarn).

**Comment by vasharma05** [comment]
Oh yes, I'll keep that in mind @FlorianRappl  !

**Pull request closed without merging**